client/lib/post-normalizer/rule-content-link-jetpack-carousels.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External Dependencies
+ */
+import { forEach, get } from 'lodash';
+
+export default function linkJetpackCarousels( post, dom ) {
+	const galleries = dom.querySelectorAll( '.tiled-gallery' );
+
+	forEach( galleries, gallery => {
+		let extra = get( gallery, [ 'dataset', 'carouselExtra' ], false );
```

Comment:
This is mostly for our test framework. jsdom doesn't support data attributes in the same way that browsers do.